File: app/assets/javascripts/models/gig.js
App.Gig = DS.Model.extend({
  name: DS.attr('string'),
  lat: DS.attr('number'),
  lng: DS.attr('number'),
  start: DS.attr('string'),
  end: DS.attr('string'),
  email: DS.attr('string'),
  notes: DS.attr('string'),
  externalUrl: DS.attr('string'),
  userIsGoing: DS.attr('boolean'),
  ticketUrl: DS.attr('string'),
  promoted: DS.attr('boolean'),
  cat1: DS.attr('string'),
  cat2: DS.attr('string'),
  pricelower: DS.attr('number'),
  pricehigher: DS.attr('number'),
  user_id: DS.attr('string')
});

File: app/models/gig.rb
class Gig < ActiveRecord::Base
  belongs_to :user
  has_many :media_files
attr_accessible :cat1, :cat2, :email, :end, :external_url, :lat, :lng, :name, :notes, :pricehigher, :pricelower, :start, :ticket_url, :user_is_going, :user_id
attr_protected :promoted
end
